README.md

```diff
@@ -365,6 +366,34 @@ The maxLength attribute added to the input. Specifies the maximum number of char
     ...>
 ```
 
+<a name="inline"></a>
+##### inline (optional)
+The inline attribute use to determine that selected tags and input field will be put in the same div block or not. If you want to separate it and apply auto scroll for just selected tags only, change this attribute. Specifies the inline attribute.
+
+```
+<ReactTags
+    inline = {false}
+    ...>
+```
+
+If inline is true (default value)
+
+```
+<div className="...">
+	{tagItems}
+	{tagInput}
+</div>
```

Comment:
I would recommend adding screen shots in both cases instead of showing how its rendered in div

Reply:
Hi @ad1992, I fix them all as request. Please help to take a look. Thanks!

Reply:
Thanks @hoantran-it  :) .The screenshots look too big and in both the case it seems that they are not inline. Can you decrease the size of screenshots may be something like this 
<img width="519" alt="screen shot 2018-03-17 at 5 20 12 am" src="https://user-images.githubusercontent.com/11256141/37549112-1f5ae3aa-29a3-11e8-971a-ecfcc629c3f0.png">
<img width="309" alt="screen shot 2018-03-17 at 5 21 34 am" src="https://user-images.githubusercontent.com/11256141/37549113-1f986dd8-29a3-11e8-832a-134ec66bd386.png">

Reply:
Hi @ad1992 can I use your screenshot, i think it's suitable :)

Reply:
sure 👍 @hoantran-it

Reply:
@ad1992 Thanks! Updated, please help to take a look :)